bimvee/importIitYarp.py

```diff
@@ -34,132 +36,208 @@
     For imu:
         - "ts": numpy array of float - seconds
 
-    Can also handle LAE (labelled event) bottles, IMU
+    Can also handle LAE (labelled event) bottles, FLOW (dvs optical flow events), 
+        IMU, SKE (skin), EAR, SKS (skin samples)
 
-This function combines the import of two distinctly different cases:
-    1) pre-split output directly from zynqGrabber
+This function combines the import of several distinctly different cases:
+    1) output directly from zynqGrabber, with all events together
+        May combine samples and events in the same bottles and never contains
+        explicit IMU / LAE / FLOW events. samples such as IMU are contained 
+        within AE type bottles.
     2) split and processed events from vPreProcess or other downstream modules
-The former may combine samples and events in the same bottles and never contains
-explicit IMU / LAE / FLOW events. samples such as IMU are contained within AE type bottles.
-The latter usually has distinct bottles for each datatype, and channels
-have probably, but not definitely, been split into left and right. 
+        The latter usually has distinct bottles for each datatype, and channels
+        have probably, but not definitely, been split into left and right. 
+    3) output from binary event dumper.
+    4) data from vicon
+    5) Skin samples which have not been properly formatted as bottles
+
 TODO: import frames - yarp has 2 ways of encoding - as bottles or,
 recommended, by saving e.g. pngs to a folder - support the latter
+
+There are 2 dvs codecs: 20-bit and 24-bit. 24-bit is assumed unless kwarg 
+"codec" = "20bit" is received
+
+This document gives more principled and complete info for event decoding
+specifically from iCub:
+https://github.com/event-driven-robotics/ed-skin/blob/master/docs/technotes/TN001-Peripherals-to-HPU-Data-Encoding_rev1.pdf
```

Comment:
The link points to a private repository. Instead, it would be better to refer to:
https://github.com/robotology/event-driven/blob/master/documentation/eventcodecs.md

However, the documentation available on robotology/event-driven should be extended to skin events/samples.